### PR TITLE
[DFMD-7] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by wuhmi**
CSS1a
CSS1b
CSS1c
CSSTabletview

### CODE REVIEW

**Comment by adaugochi** [comment]
Well dome. Kindly send a mobile screenshot of your work

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+   /* selected link */
+ a:active {
+     text-decoration: underline;
+     color: burlywood;
+ }
+/* end of link styling */
+ 
+
+/* Footer styling */
+.FOOTER {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:72]: This should not be in block/capital letters.

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    width: 300px;
+}
+
+img.SHMPOO {
+    width: 400px;
+    height: 190px;
+    border: 2px solid wheat;
+}
+
+.Specs {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:148]: Your class naming should start with a small letter

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    border: 4px solid wheat;
+    border-radius: 3px;
+}
+
+h4 {
+    text-align: center;
+    font-size: 15px;
+}
+
+.IMG1,
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:174]: It's not a good practice to use uppercase in your naming convention. I think you should also use something more descriptive for your selectors.

Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:174]: It's not a good practice to use uppercase in your naming convention. I think you should also use something more descriptive for your selectors.

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 2 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:148]: Your class naming should start with a small letter

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 1 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:72]: This should not be in block/capital letters.

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+.IMG2,
+.IMG3 {
+    float: left;
+    display: inline;
+    width: 33.3%;
+}
+
+.SL,
+.VM,
+.EG {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:184]: Same here hoo

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    position: relative;
+    left: 50px;
+    text-align: center;
+}
+
+img.SOIL,
+img.VMOSE,
+img.GEL {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:192]: Here too

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+    display: table;
+    clear: both;
+}
+
+
+/* Media Responsiveness */
+@media only screen and (max-width: 600px) {
+    .IMG1,
+    .IMG2,
+    .IMG3 {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:207]: Here too

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 1 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:72]: This should not be in block/capital letters.

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 2 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:148]: Your class naming should start with a small letter

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+.Specs {
+    float: right;
+    display: inherit;
+    overflow: hidden;
+    margin: 0;
+    padding: 35px 16px;
+    width: 58%;
+}
+
+#Quantity {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:157]: same here too

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+}
+
+h4 {
+    text-align: center;
+    font-size: 15px;
+}
+
+.IMG1,
+.IMG2,
+.IMG3 {
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:176]: It's not a good practice to use uppercase in your naming convention. I think you should also use something more descriptive for your selectors.

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 4 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:184]: Same goes here too

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 5 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:192]: Here too

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css (hunk 6 above)
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:207]: Here too

File: submissions/omawumiRebeccaMoju/HTML/productpage.html [hunk 9]
```diff
@@ -1,72 +1,102 @@
...
+    <link href="../CSS/styleproductpage.css" rel="stylesheet" type="text/css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
 </head>
 
 <body>
-    <h1>BLK_ My Naturals</h1</a>
-        <nav>
-            <div>
+    <header>
+        <div class="nav bar">
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/HTML/productpage.html:14]: `bar` was not used anywhere in your CSS code.

File: submissions/omawumiRebeccaMoju/HTML/productpage.html [hunk 10]
```diff
@@ -1,72 +1,102 @@
...
-                        Our products are made from strictly organic ingredients which helps give our naturals the
-                        healthy look it deserves.
-                        We have different products according to the different needs of the black natural hair.
-                        BLK helps you wear your natural curls like the Queen you are.
+            </nav>
+        </div>
+    </header>
+    <br />
+    <main>
+        <img class="SHMPOO" src="Images/N-SHAMPOO.jpg" controls></Images>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/HTML/productpage.html:27]: Please ensure you use a proper naming convention for your classes.

File: submissions/omawumiRebeccaMoju/HTML/productpage.html [hunk 11]
```diff
@@ -1,72 +1,102 @@
...
+        <div class="Specs">
+            <p>
+                <strong>BLK NOURISHING SHAMPOO: 250ml</strong><br>
+                <strong class="QTY">Quantity:</strong>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/HTML/productpage.html:31]: Same here too

File: submissions/omawumiRebeccaMoju/HTML/productpage.html [hunk 12]
```diff
@@ -1,72 +1,102 @@
...
+            itchy scalp while effectively removing dirt and product build-up. Wash hair with ample amount of BLK_My
+            Naturals Nourishing
+            Shampoo alongsides BLK_My Naturals Nourishing Conditioner to attain best results. Then, dry.
+        </p>
+        <hr>
+        <div class="products">
+            <h4>SIMILAR PRODUCTS</h4>
+            <ul>
+                <div class="IMG1">
+                    <img class="SOIL" src="Images/S-OIL.jpg" controls>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/HTML/productpage.html:51]: Here too

File: submissions/omawumiRebeccaMoju/HTML/productpage.html [hunk 13]
```diff
@@ -1,72 +1,102 @@
 <!DOCTYPE html>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/HTML/productpage.html:1]: Please take time to make corrections and changes to this HTML file

File: submissions/omawumiRebeccaMoju/html-forms/productpage.html [hunk 14]
```diff
@@ -35,27 +41,58 @@ <h2>About BLK_My Naturals</h2>
...
+            Color: Gold
+            <br/>
+            <label for="Quantity"><strong>Quantity:</strong></label>
+            <input type="number" name="Quantity" id="Quantity">
+            <br/>
+            Price: &#x20A6; 3,500<br/>
+            <button class="btn" type="button">BUY NOW</button>
+    </div>
+        <p> 
+            <h4><strong>PRODUCT DESCRIPTION</strong></h4>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/html-forms/productpage.html:55]: An `h` tag cannot go inside a `p` tag. Kindly separate them.

File: submissions/omawumiRebeccaMoju/html-forms/productpage.html [hunk 15]
```diff
@@ -35,27 +41,58 @@ <h2>About BLK_My Naturals</h2>
...
+        <input type="text" name="E-mail" id="E-mail" placeholder="Enter your email address">
+        <button type="button">SUBMIT</button>
+        </div>
+        <br />
+        <div class="sites">
+        <a href="https://www.facebook.com">Facebook</a>
+        <a href="https://www.twitter.com">Twitter</a>
+        <a href="https://www.G-mail.com">G-mail</a>
+        <a href="https://www.youtube.com">Youtube</a>
+        </div>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/html-forms/productpage.html:95]: Format properly

File: submissions/omawumiRebeccaMoju/html-forms/productpage.html [hunk 16]
```diff
@@ -1,24 +1,30 @@
 <!DOCTYPE html>
```
Comment (TundeOladejo) [inline submissions/omawumiRebeccaMoju/html-forms/productpage.html:1]: Change the name of this file to something else. You already have a file with that name already

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

**Comment by wuhmi** [comment]
MOBILE1
MOBILE 2
MOBILE 3
MOBILE 4
MOBILE 5

**Comment by wuhmi** [comment]
MOBILE 4
MOBILE 5

File: submissions/omawumiRebeccaMoju/CSS/styleproductpage.css [hunk 17]
```diff
@@ -0,0 +1,260 @@
...
+.img-vmose,
+.img-gel {
+    float: left;
+    display: inline;
+    width: 33.3%;
+}
+
+.sl,
+.vm,
+.eg {
```
Comment (adaugochi) [inline submissions/omawumiRebeccaMoju/CSS/styleproductpage.css:183]: These `sl, vm, eg`  are not good class names. Kindly give it a better name. Also, no need to assign different class names. Just give it a single class name and use that class name all other the place you want to use it in your HTML code

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**